frontend/src/js/prime-web-receiver-pre.js

```diff
@@ -184,7 +184,14 @@ function requestFile(reportId) {
 
     return window.jwt? axios.get(`${baseURL}/api/history/report/${reportId}`, apiConfig())
         .then(res => res.data)
-        .then(csv => download(csv.content, csv.filename, csv.mimetype)) : null;
+        .then(csv => {
```

Comment:
Does this mean that any call to `api/history/report/${reportId}` will return a path that cannot be downloaded just like that unless you 'fix' the path? If so, can we fix it server side instead to fix it for everyone? (unless I'm misunderstanding)

Reply:
The API will return the blob name.  In the case of this code, the blob name will include the folder names as `ready%2Faz-phd.elr-prod%2Faz-covid-19-7edf57cf-a6d4-4a3f-98bb-300a2f9a0018-20210714141108.csv`.  I am worried about someone needing the blob name to do other operations, so not sure if removing the folders at the API level could hurt any future applications.

Reply:
Ok, this make sense